### DISCUSSION TITLE
I got PermissionError: [Errno 13] Permission denied

### DISCUSSION THREAD

**Discussion by TT9x** [voicepaw/so-vits-svc-fork#693] [Q&A]
After gui loaded, i load model, mp3 file and click Start Voice Changer but nothing happen and i got this error on console window
![image](https://github.com/voicepaw/so-vits-svc-fork/assets/123978112/8c1160e3-f087-4429-a1fe-1496aaaedec4)

**Comment by idnotbe** [comment]
i have the same issue too, when i did with a RVC file.
but with a 4.0 file, it works.

what is the difference between RVC and a normal 4.0??

**Reply by evilcrusher2** [reply]
RVC is an entirely different method of AI voice cloning software. The config files are much different and do not work with so-vits

**Comment by willtraven** [comment]
i'm also getting this problem... and second that question from idnotbe. Also... how can we tell if a file is a 4.0 file? like are they labeled somewhere?
